### PR TITLE
fixing auth edge case

### PR DESCRIPTION
**Pull request by jmervine**
Fixing https://rollbar.com/Heroku-3/retrodot/items/90/

### CODE REVIEW

**Comment by sheax0r** [comment]
👍

File: spec/helpers/active_admin/views_helper_spec.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 RSpec.describe ActiveAdmin::ViewsHelper do
-  describe :format_duration do
+  describe "format_duration" do
```
Comment (jmervine) [inline spec/helpers/active_admin/views_helper_spec.rb]: Something in the `rails-controller-testing` gem doesn't allow symbols as describe arguments.

File: spec/controllers/application_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+require "spec_helper"
+
+RSpec.describe ApplicationController, type: :controller do
+  describe "#current_user" do
+    let(:invalid) { create(:user, handle: nil) }
+    let(:valid)   { create(:user) }
+
+    before do
+      # unstub current_user
+      allow(subject).to receive(:current_user).and_call_original
```
Comment (sheax0r) [inline spec/controllers/application_controller_spec.rb]: Why do we need this? Is there some sort of override of the `current_user` method in spec helpers..?

Comment (jmervine) [inline spec/controllers/application_controller_spec.rb]: This simple resets `current_user`. It shouldn't be required, but I'm pretty sure (when building the spec I copied this from months ago) I was getting odd errors with stuck stubs.

**Pull request merged**